### PR TITLE
Margins and paddings: Add new auto generation options

### PR DESCRIPTION
**Pull request by Pineirin**
closes: https://github.com/inveniosoftware/invenio-administration/issues/61

Screenshot from 2022-09-13 17-07-38

### CODE REVIEW

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/menu.overrides [hunk 1]
```diff
@@ -61,6 +61,10 @@
       border-radius: 0;
       background-color: transparent;
 
+      .ui.dropdown.custom-plus-menu {
```
Comment (jrcastro2) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/collections/menu.overrides:64]: Can you double check if this is needed, maybe it's enough by using the already created classes `rel-b{alias}-{value}` ?

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 2]
```diff
@@ -79,126 +79,113 @@ code {
...
 * }
-* .mt-{5,10,15,20,25,30} {
+* .{rel-m,rel-p}t-{0,1,2,3,4,5} {
 *     margin-top {
-*       {5,10,15,20,25,30}px !important;
+*       {1,2,3,4,5}em !important;
 *     }
 * }
-* .mb-{5,10,15,20,25,30} {
+* .{rel-m,rel-p}b-{0,1,2,3,4,5} {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:159]:
```suggestion
* .{rel-m,rel-p}b-{1,2,3,4,5} {
```

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 3]
```diff
@@ -79,126 +79,113 @@ code {
...
+*     }
+* }
+* .{rel-m,rel-p}l-{0,1,2,3,4,5} {
 *     margin-left {
-*       {5,10,15,20,25,30}px !important;
+*       {1,2,3,4,5}em !important;
 *     }
 * }
-* .mt-{5,10,15,20,25,30} {
+* .{rel-m,rel-p}t-{0,1,2,3,4,5} {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:154]:
```suggestion
* .{rel-m,rel-p}t-{1,2,3,4,5} {
```

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 4]
```diff
@@ -79,126 +79,113 @@ code {
...
+*       {0,5,10,15,20,25,30}px !important;
 *     }
 * }
-* .ml-{5,10,15,20,25,30} {
+* .{m,p}b-{0,5,10,15,20,25,30} {
+*     margin-bottom {
+*       {0,5,10,15,20,25,30}px !important;
+*     }
+* }
+* .{rel-m,rel-p}-{0,1,2,3,4,5} {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:139]:
```suggestion
* .{rel-m,rel-p}-{1,2,3,4,5} {
```

File: invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides [hunk 5]
```diff
@@ -79,126 +79,113 @@ code {
...
+*     {margin, padding} {
+*       {1,2,3,4,5}em !important;
+*     }
+* }
+* .{rel-m,rel-p}r-{0,1,2,3,4,5} {
```
Comment (kpsherva) [inline invenio_theme/assets/semantic-ui/less/invenio_theme/theme/globals/site.overrides:144]:
```suggestion
* .{rel-m,rel-p}r-{1,2,3,4,5} {
```

**Pull request merged**